week7/cooper_client_react/01_cooper_calculator.md

```diff
@@ -0,0 +1,398 @@
+# Cooper client in React
+
+First we need to create the application. Run:
```

Comment:
Is this the first file in this chapter? If so I would need you to have a more extensive introduction to the challenge, learning objectives, background, etc.

Reply:
Yes it is, I will get on it.